packages/components/Checkbox/SPEC.md

```diff
@@ -256,14 +325,20 @@ A disabled `Checkbox` is non-interactive, disallowing the user to click/tap on i
 
 A hovered `Checkbox` changes styling to communicate that the user has placed a cursor above it. On win32, this will show a checkmark in a checkbox.
 
+Note: Not Supported on Android
```

Comment:
And iOS, would have preferred `unsupported on mobile`

Reply:
Addressed on https://github.com/microsoft/fluentui-react-native/pull/2416